src/rc/mod.rs

```diff
@@ -16,10 +16,13 @@ let mut generator = Gen::new(producer);
 
 ## Using 'proc_macro`
 
-A macro attribute can be used for functions [`rc_yield_fn`](macros.rc_yield_fn.html), and a function like
-macro for closures [`rc_yield_cls`](macros.rc_yield_cls.html). These are meant to be used
-with the [`yield_'](macros.yield_.html) macro for easy definition of generators.
-
+A macro attribute can be used for functions `rc_yield_fn`, and a function like macro
+for closures `rc_yield_cls`. These are meant to be used with the `yield_` macro for
+easy definition of generators. The crate must be compiled with the `proc_macro`
+feature for these to be enabled.
+```toml
+syn = {version = "0.2", features = ["proc_macro"] }
```

Comment:
I added an example of possible documentation in one of the mods let me know what you think. Is there a way to conditional test the docs these fail some of the tests, should all tests run with all features?

Reply:
Pretty good. I would take the prose and move it to the main module (so you're not copy/pasting text across all three modules) and leave just minimal code examples in the examples sections.

If you want, I'd be happy to handle writing the docs for this after we get this merged so you don't have to worry about it

Reply:
Yeah that'd be nice, keep it one consistent voice and way of explaining things. Cool that'd be nice thanks.